src/flameGraph.h

```diff
@@ -41,15 +47,19 @@ class Trie {
     }
 
     Trie* child(u32 name_index, FrameTypeId type) {
-        return &_children[name_index | type << 28];
+        auto ptr = &_children[name_index | type << 28];
```

Comment:
It's perfectly fine to use `auto` for iterators for the sake of brevity, but please do not overuse it.
Types are very important for understanding code, even more than names.

Reply:
Sure, fixed in 0885f458a42744238df5d2f3408042837d1f4e2c